consuming-webservices/Networking/RequestHandler.swift

```diff
@@ -45,12 +45,44 @@ struct HTTPRequestHandler: RequestHandler {
     var body: Any?
     
     func execute( callback: @escaping (Result<Any>) -> Void) {
-        var returnValue = 0
-        if let body = body as? [String: String] {
-            returnValue = body["args"] == "on" ? 1 : 0
+        guard let url = URL(string: path) else { return }
```

Comment:
It would be best to call the callback and pass it a `.failure(RequestError.invalidURL)` before the return. I realize I gave you that guard statement, but I didn't include this detail for the sake of brevity.